package.json

```diff
@@ -256,6 +256,12 @@
     "css-what": ">= 5.0.1",
     "trim-newlines": ">= 3.0.1",
     "glob-parent": ">= 5.1.2",
-    "ws": ">= 7.4.6"
+    "ws": ">= 7.4.6",
+    "**/node-forge": "1.0.0",
+    "**/axios": "0.21.2",
+    "**/postcss": "8.2.13",
+    "**/json-schema": "0.4.0",
+    "**/node-fetch": "2.6.7",
+    "**/follow-redirects": "1.14.8"
```

Comment:
what does this do?

Reply:
The resolutions section overrides the dependencies pulled into yarn.lock.

Lets look at node-forge

Prior to this, something dragged in a dependency that resolved to node-forge 0.10.0 (selfsigned is one package that uses it). The above is telling yarn (lerna?) to use node-forge 1.0.0 in yarn.lock.

I've added a couple of comments below.

The others are doing the same thing. If the tests pass we should be okay. ideally we would update the top level package that indirectly references it and that would update all of the offending dependencies. Unfortunately its never that easy.